egs/wsj/s5/steps/dict/select_prons_bayesian.py

```diff
@@ -0,0 +1,423 @@
+#!/usr/bin/env python
+
+# Copyright 2016  Xiaohui Zhang
+# Apache 2.0.
+
+from __future__ import print_function
+from collections import defaultdict
+import os
+import argparse
+import sys
+import warnings
+import copy
+import imp
+import math
+import numpy
+
```

Comment:
Samuel, I see a bunch of modules you are not actually using here and in the other  python scripts.
Please remove those.  Also, as a general principle, if you importing something outside the default python libs, you should be little bit more helpful with what is wrong.

i.e.
try: 
  import numpy
except ImportError:
  print "Error: you don't seem to have numpy installed, OR the install is not functional"
  raise

btw, please install flake8 and run your script through that program